test/cmd/test-scenario/generator/suite_test.go

```diff
@@ -1,4 +1,4 @@
-package main
+package generator
```

Comment:
```suggestion
package generator_test
```

Isn't it the go way?

Reply:
Plus perhaps we could name the file `generator_suite_test.go`

Reply:
Line 16: `RunSpecWithJUnitReporter(t, "Session reconciler Suite")` - copy&pastie? :)

Reply:
I would argue for having it named "suite_test.go" since it's already package scoped to avoid the stutter. But I'm fine with consistency across code base and rather rename it back together with all other files as well at some later point.

Reply:
https://github.com/Maistra/istio-workspace/pull/386/commits/c2fc1bdc0b05a205335c3e8db671727fcada2b9d

Reply:
@bartoszmajsak Yes, copy&paste 8 months ago hehe. The file is only renamed now, but fixed in commit above. :)

Reply:
Yeah I know, someone made a sloppy review back then xD

Reply:
It's still `package generator` though

Reply:
Ah yes.. forgot the suite package. :)